backend/src/src-wsimport/src/main/java/org/eclipse/sw360/wsimport/rest/WsImportService.java

```diff
@@ -70,4 +75,20 @@ public WsProject getWsProject(String projectToken, TokenCredentials tokenCredent
 
     }
 
+    public  WsProjectVitalInformation[] getOrganizationalProjectVitals(TokenCredentials tokenCredentials) throws JsonSyntaxException {
+        String projectVitalString;
+        try {
+            projectVitalString = restClient.getData(REQUEST__GET_ORGANIZATION_PROJECT_VITALS, tokenCredentials.getToken(), WsTokenType.ORGANIZATION, tokenCredentials);
+        } catch (IOException ioe) {
+            LOGGER.error(ioe);
```

Comment:
Please add your error message to the logger output, like logger.error("can not get data from..", ioe");

Reply:
👍